aikit/datasets/datasets.py

```diff
@@ -36,21 +40,72 @@ def get_dataset_config():
     config_file = os.path.join(os.path.split(__file__)[0], "config.json")
     if not os.path.exists(config_file):
         return None
-
     return load_json(config_file)
 
 
 DATASET_CONFIG = get_dataset_config()
 
 
-# from aikit.helper_functions import save_json
-
-
-def find_path(name):
+DATASET_PUBLIC_URLS = {
+    'titanic': 'https://github.com/gfournier/aikit-datasets/releases/download/titanic-1.0.0/titanic.tar.gz'
+}
+
+
+def _load_public_path(url, cache_dir=None, cache_subdir='datasets'):
+    """
+    Load a public dataset from the specified URL. The data is loaded locally in the cache directory
+
+    # Arguments
```

Comment:
Need to use sklearn like docstring:
Parameters
------------

Reply:
https://github.com/societe-generale/aikit/pull/2/commits/e2729f328b7a3086a908ed270f82269173812cc8